spec/cucumber_factory/steps_spec.rb

```diff
@@ -282,4 +282,58 @@
     invoke_cucumber_step('there is a plain ruby class with the butt "pear-shaped"')
   end
 
+  it "should should allow to set attributes via doc string" do
+    user = User.new
+    User.stub(:new => user)
+
+    invoke_cucumber_step(<<-DOC_STRING)
```

Comment:
You need to change the `invoke_cucumber_step` method to properly pass a multiline string like cucumber would do it. You'll see what to do when you take a look at that method.